blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -317,12 +258,13 @@ const Nav = (props) => {
             key={`${id}_${breakpoint}_${i}`}
           >
             <NavWidget
-              {...props}
```

Comment:
using children props explicitly here so that it's more apparent how this is working